backend/src/src-licenses/src/main/java/com/siemens/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -280,17 +352,39 @@ private void filterTodoWhiteList(String organisation, List<Todo> todos) {
 
 
     public static <T> List<T> getEntriesFromIds(final Map<String, T> map, Set<String> ids) {
-        return FluentIterable.from(ids).transform(new Function<String, T>() {
-            @Override
-            public T apply(String input) {
-                return map.get(input);
+        return ids
+                .stream()
+                .map(input -> map.get(input))
+                .filter(input -> input !=null)
+                .collect(Collectors.toList());
+    }
+
+    public RequestStatus updateLicense(License moderatedLicense, User user, User requestingUser) {
+        if (PermissionUtils.isClearingAdmin(user)) {
```

Comment:
This excludes the admin? I think it would be better to use `PermissionUtils.isUserAtLeast(...)`